feature-utils/poly-look/test/react-components/popUpBanner.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { PopUpBanner } from "../../src/react-components";
+
+/**
+ * @jest-environment jsdom
+ */
+
+const children = "Pop Up content";
+const reportSuccessful = "successful";
+const reportUnsuccessful = "unsuccessful";
+
+it("PopUpBanner renders correctly", () => {
```

Comment:
Maybe put it inside a `describe` statement, simply for symmetry.

Reply:
Do you mean having two `describe`?

Reply:
Yep